doc/reference_cluster.xml

```diff
@@ -393,7 +465,75 @@ GEOMETRYCOLLECTION(LINESTRING(6 6,7 7))
         <para>
           <xref linkend="ST_ClusterDBSCAN" />,
           <xref linkend="ST_ClusterKMeans" />,
-          <xref linkend="ST_ClusterIntersecting" />
+          <xref linkend="ST_ClusterIntersectingWin"/>,
+          <xref linkend="ST_ClusterWithinWin"/>,
+          <xref linkend="ST_ClusterIntersecting"/>
+        </para>
+      </refsection>
+
+    </refentry>
+
+  <refentry id="ST_ClusterWithinWin">
+      <refnamediv>
+        <refname>ST_ClusterWithinWin</refname>
+
+        <refpurpose>Window function that returns a cluster id for each input geometry, clustering using separation distance.</refpurpose>
+      </refnamediv>
+
+      <refsynopsisdiv>
+        <funcsynopsis>
+          <funcprototype>
+            <funcdef>integer <function>ST_ClusterWithinWin</function></funcdef>
+            <paramdef><type>geometry winset </type> <parameter>geom</parameter></paramdef>
+            <paramdef><type>float8 </type> <parameter>distance</parameter></paramdef>
+          </funcprototype>
+        </funcsynopsis>
+      </refsynopsisdiv>
+
+      <refsection>
+        <title>Description</title>
+
+        <para>ST_ClusterWithinWin is a window function that returns an integer for each input geometry, where the integer the cluster number the geometry is a member of. Clusters represent a set of input geometries separated by no more than the specified distance. (Distances are Cartesian distances in the units of the SRID.)
+        </para>
+
+        <para>Availability: 3.4.0</para>
+      </refsection>
```

Comment:
```suggestion
      <para>&curve_support;</para>
      </refsection>
```

Reply:
By the same token should ClusterIntersecting have this marker? I'm not sure if it means "native arc support" (like distance and area have) or "stroked arc support" (like all the GEOS backed functions have).

Reply:
I guess I would interpret it to mean native arc support, otherwise all functions would have arc support, no? (But I agree, the wording isn't clear one way or the other.) In this case ClusterWithin would have native arc support but ClusterIntersecting uses GEOS so it would not.

Reply:
DBScan is not currently marked as having arc support. add there too?